### PR TITLE
Add readme notes about license and who maintains

### PR DESCRIPTION
**Pull request by ktoso**
As agreed in the "reqs to be under lightbend/... github".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,14 @@ reported to [github issue tracker](https://github.com/lightbend/dbuild/issues).
...
 
 see [CHANGELOG.md](CHANGELOG.md)
+
+## License
+
+This software is licensed under the Apache 2 license.
+
+### Developed by Lightbend
+
+Maintained by [Scala Team](https://github.com/orgs/lightbend/teams/scala)
```
Comment (dwijnand) [inline README.md]: Tooling Team?

Comment (SethTisue) [reply README.md]: yes :-)

**Comment by SethTisue** [comment]
should this also say it's not covered by Lightbend support?

**Comment by cunei** [comment]
There is some duplication, please note https://github.com/lightbend/dbuild/blame/master/README.md#L15

**Comment by dwijnand** [comment]
Personally I think that should be detailed elsewhere, without whatever wording necessary to clarify what to expect to be and not be included in support; rather than the current approach of (semi-arbitrarily) defining that a project is (for example) not included in support.

**Comment by SethTisue** [comment]
“unsupported” is ambiguous about whether it refers to commercial Lightbend support in particular, or just “support” in general. but whatevs, this is mergeable as-is I think.

**Comment by ktoso** [comment]
We can be explicit about "not supported under lightbend subscription" I guess in projects, but this seems fine,

**Pull request merged**